src/SearchIndex/index.ts

```diff
@@ -266,3 +267,23 @@ const composeUpserts = (
     };
   });
 };
+
+// similar to atlas admin getFacetKeys but this one is not
```

Comment:
is it different? CMIIW but getting the same list of keys from the function mentioned in this line